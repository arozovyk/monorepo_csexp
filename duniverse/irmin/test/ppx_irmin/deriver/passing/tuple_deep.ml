(* Nested tuple type *)
type deep_tuple =
  (((int32 * int32) * int32 * int32) * int32 * int32) * int32 * int32
[@@deriving irmin]

let (_ : deep_tuple Irmin.Type.t) = deep_tuple_t
