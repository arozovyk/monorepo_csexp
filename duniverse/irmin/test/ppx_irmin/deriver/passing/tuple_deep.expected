type deep_tuple =
  ((((int32 * int32) * int32 * int32) * int32 * int32) * int32 * int32)
[@@deriving irmin]
include
  struct
    let deep_tuple_t =
      let open Irmin.Type in
        triple (triple (triple (pair int32 int32) int32 int32) int32 int32)
          int32 int32
  end[@@ocaml.doc "@inline"][@@merlin.hide ]
let (_ : deep_tuple Irmin.Type.t) = deep_tuple_t
